### ISSUE TITLE
bug no-unsupported-browser-features with true value

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#870]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Set `no-unsupported-browser-features` to `true` throw error.

> Which rule, if any, is this issue related to?

e.g. `no-unsupported-browser-features`

> What CSS is needed to reproduce this issue?

no matter

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "no-unsupported-browser-features": true
  }
}
```

> Which version of stylelint are you using?

e.g. `4.5.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Run without error

> What actually happened (e.g. what warnings or errors you are getting)?

Throw error `Cannot read property 'ignore' of undefined`

```shell
[20:56:04] Plumber found unhandled error:
 TypeError in plugin 'gulp-postcss'
Message:
    Cannot read property 'ignore' of undefined
```

**Comment by jeddy3** [comment]
Ha, @evilebottnawi you were even quicker reporting a issue on this release than the last one! :)

Thanks, as always, for reporting. Pushing up a PR for this one now.

**Comment by jeddy3** [comment]
@evilebottnawi FYI, released in `4.5.1`
